mod-data-import-converter-storage-server/src/main/resources/templates/db_scripts/defaultData/default_qm_authority_update_job_profile.sql

```diff
@@ -0,0 +1,190 @@
+	"deleted": false,
+	"hidden": true,
+	"dataType": "MARC",
+	"metadata": {
+		"createdDate": "2022-04-19T09:07:47.667",
+		"updatedDate": "2022-04-19T09:09:10.382+0000",
+		"createdByUserId": "6a010e5b-5421-5b1c-9b52-568b37038575",
+		"updatedByUserId": "6a010e5b-5421-5b1c-9b52-568b37038575"
+	},
+	"userInfo": {
+		"lastName": "System",
+		"userName": "System",
+		"firstName": "System"
+	},
+	"childProfiles": [],
+	"parentProfiles": []
+}
+') ON CONFLICT DO NOTHING;
+
+INSERT INTO ${myuniversity}_${mymodule}.action_profiles (id, jsonb) values
+('f0f788c8-2e65-4e3a-9247-e9444eeb7d70',
+'{
+	"id": "f0f788c8-2e65-4e3a-9247-e9444eeb7d70",
+	"name": "quickMARC - Default Update MARC authority",
+	"action": "UPDATE",
+	"deleted": false,
+	"hidden": true,
+	"metadata": {
+		"createdDate": "2020-11-30T09:02:39.96",
+		"updatedDate": "2020-11-30T11:57:24.083+0000",
+		"createdByUserId": "6a010e5b-5421-5b1c-9b52-568b37038575",
+		"updatedByUserId": "6a010e5b-5421-5b1c-9b52-568b37038575"
+	},
+	"userInfo": {
+		"lastName": "System",
+		"userName": "System",
+		"firstName": "System"
+	},
+	"description": "Updates existing SRS MARC authority record",
+	"folioRecord": "MARC_BIBLIOGRAPHIC",
+	"childProfiles": [],
+	"parentProfiles": []
+}
+') ON CONFLICT DO NOTHING;
+
+
+INSERT INTO ${myuniversity}_${mymodule}.match_profiles (id, jsonb) values
+('aff72eae-847c-4a97-b7b9-c1ddb8cdcbbf',
+'{
+	"id": "aff72eae-847c-4a97-b7b9-c1ddb8cdcbbf",
+	"name": "quickMARC - Default match for existing SRS authority",
+	"deleted": false,
+	"hidden": true,
+	"metadata": {
+		"createdDate": "2020-11-30T09:06:01.52",
+		"updatedDate": "2020-11-30T09:59:01.248+0000",
+		"createdByUserId": "6a010e5b-5421-5b1c-9b52-568b37038575",
+		"updatedByUserId": "6a010e5b-5421-5b1c-9b52-568b37038575"
+	},
+	"userInfo": {
+		"lastName": "System",
+		"userName": "System",
+		"firstName": "System"
+	},
+	"description": "Matches the UUID from the 999 ff $s in the incoming MARC record to the same field in any SRS MARC authority",
+	"matchDetails": [{
+			"matchCriterion": "EXACTLY_MATCHES",
+			"existingRecordType": "MARC_BIBLIOGRAPHIC",
+			"incomingRecordType": "MARC_BIBLIOGRAPHIC",
+			"existingMatchExpression": {
+				"fields": [{
+						"label": "field",
+						"value": "999"
+					}, {
+						"label": "indicator1",
+						"value": "f"
+					}, {
+						"label": "indicator2",
+						"value": "f"
+					}, {
+						"label": "recordSubfield",
+						"value": "s"
+					}
+				],
+				"dataValueType": "VALUE_FROM_RECORD"
+			},
+			"incomingMatchExpression": {
+				"fields": [{
+						"label": "field",
+						"value": "999"
+					}, {
+						"label": "indicator1",
+						"value": "f"
+					}, {
+						"label": "indicator2",
+						"value": "f"
+					}, {
+						"label": "recordSubfield",
+						"value": "s"
+					}
+				],
+				"dataValueType": "VALUE_FROM_RECORD"
+			}
+		}
+	],
+	"childProfiles": [],
+	"parentProfiles": [],
+	"existingRecordType": "MARC_BIBLIOGRAPHIC",
```

Comment:
I suppose this should be "MARC_AUTHORITY", isn't it?
Same for other places in authorities, also seams like same situation in holdings

Reply:
oh, I see. Got it thank u